src/main/java/org/commcare/core/graph/c3/AxisConfiguration.java

```
@@ -118,10 +121,21 @@ private void addTickConfig(JSONObject axis, String key, String varName) throws G
                         } else {
                             values.put(parseDouble(location, key));
                         }
+                        try {
+                            String current = labels.getString(location);
+                            if (current.length() > largestLabel.length()) {
+                                largestLabel = current;
+                            }
+                        } catch (JSONException e) { }

```

should we throw an error here ?